reference/import_export/connectors.rst

```diff
@@ -0,0 +1,29 @@
+Akeneo Native Connector
+=======================
+
+CSV Format
+----------
+
+Akeneo Native Connector supports reading and writing CSV files.
+
+XLSX Format
+-----------
+
+Akeneo Native Connector supports reading and writing XLSX files. The connector uses an open source library called Spout (https://github.com/box/spout) to read and create XLSX files.
+
+The connector has been tested and works with the following Microsoft Office versions:
+
+- Windows Excel 2016
+- Windows Excel 2013
+- Windows Excel 2010
+- Windows Excel 2007
+- Excel 16 (Office 2016 for OS X)
+- Excel 14 (Office 2011 for OS X)
+
+Customizing connectors
+----------------------
+
+It is possible to customize this connector to import/export numerous PIM related objects:
+
+- How to import PIM objects using this connector (cf :doc:`reference/import_export/simple-import.rst`)
```

Comment:
@nidup Is it okay if I add it now in my PR though @pierallard's PR is not merge yet ?